go.mod

```diff
@@ -67,6 +67,7 @@ require (
 	github.com/klauspost/asmfmt v1.3.2 // indirect
 	github.com/klauspost/compress v1.16.7 // indirect
 	github.com/klauspost/cpuid/v2 v2.2.4 // indirect
+	github.com/lib/pq v1.10.9 // indirect
```

Comment:
we shouldn't introduce another library to connect to Postgres — please use `github.com/jackc/pgx/v5` instead

Reply:
done.